components/forms/ltft/LtftStatusDetails.tsx

```diff
@@ -37,30 +37,6 @@ export const LtftStatusDetails = (formData: LtftObj) => {
             {(formData.status?.current?.state === "UNSUBMITTED" ||
               formData.status?.current?.state === "WITHDRAWN") && (
               <>
-                <SummaryList.Row>
```

Comment:
@EdwardBarclay  As discussed at today's stand-up, instead of deleting the whole section, you could refactor it to something like this to still allow the user who e.g. unsubmitted the application without naming an shaming them 😁 :

 ```
            <SummaryList.Row>
                  <SummaryList.Key>
                    {StringUtilities.capitalize(
                      formData.status?.current?.state
                    )}{" "}
                    by
                  </SummaryList.Key>
                  <SummaryList.Value data-cy="ltftModifiedBy">
                    {formData.status.current.modifiedBy.role === "TRAINEE"
                      ? "me"
                      : "TIS Admin"}

```

Reply:
I am currently refactoring the BE to bring through the role as currently it is removed from the modified by, I will then update this to these specifications

Reply:
Ah Ok 👍